packages/appstore/src/components/main-title-bar/__tests__/asset-summary.spec.tsx

```diff
@@ -0,0 +1,143 @@
+import React from 'react';
+import AssetSummary from '../asset-summary';
+import { render, screen } from '@testing-library/react';
+import { StoreProvider, mockStore } from '@deriv/stores';
+import { isMobile } from '@deriv/shared';
+
+jest.mock('@deriv/shared', () => ({
+    ...jest.requireActual('@deriv/shared'),
+    isMobile: jest.fn(() => false),
+}));
+
+jest.mock('Components/pre-loader/total-assets-loader', () => ({
+    __esModule: true,
+    default: () => <div>TotalAssetLoader</div>,
+}));
+
+jest.mock('@deriv/components', () => ({
+    ...jest.requireActual('@deriv/components'),
+    Popover: jest.fn(props => (
+        <>
+            <span>{props.message}</span>
+            <span>{props.alignment}</span>
+        </>
+    )),
+}));
+
+describe('AssetSummary', () => {
+    it('should render asset summary', () => {
+        const mock = mockStore({});
```

Comment:
i see duplicates mockStore used inside the test case can we move this `const mock = mockStore({})` out from the test case and make it a global const value?